src/main/java/org/sagebionetworks/web/client/place/DataAccessManagementPlace.java

```diff
@@ -1,13 +1,21 @@
 package org.sagebionetworks.web.client.place;
 
+import com.google.gwt.place.shared.Place;
 import com.google.gwt.place.shared.PlaceTokenizer;
 import com.google.gwt.place.shared.Prefix;
 
-public class DataAccessManagementPlace extends ParameterizedPlace {
```

Comment:
ParameterizedPlace was modifying the URL making it impossible to navigate directly to a subpage (e.g. #!DataAccessManagementPlace:default/Submissions)

Reply:
Perhaps unrelated, but why is this not just #!DataAccessManagementPlace:Submissions ?